extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcConfig.java

```diff
@@ -54,13 +52,25 @@
     /**
      * If set to true, Arc will transform the bytecode of beans containing methods that need to be proxyable
      * but have been declared as final. The transformation is simply a matter of removing final.
-     * This ensures that a proxy can be created properly.
+     * This ensures that a proxy/subclass can be created properly.
      * If the value is set to false, then an exception is thrown at build time indicating
      * that a proxy could not be created because a method was final.
+     * 
+     * @deprecated This config property is ignored and will be removed at some point post Quarkus 1.4
+     * @see #transformUnproxyableClasses
      */
+    @Deprecated
```

Comment:
```suggestion
    @Deprecated(forRemoval=true)
```

Reply:
Hm, `forRemoval` is JDK9+. Is it required to build quarkus? Our pom still targets 1.8... and I'm still using JDK8 ;-).

Reply:
Ah damn it... You are right I just switched to JDK 11 in my IDE and didn't realize that this was a JDK 8+ thing since I was so used to viewing JDK 8 sources :)